modules/auxiliary/admin/kerberos/get_ticket.rb

```diff
@@ -91,7 +91,7 @@ def initialize(info = {})
 
   def validate_options
     if datastore['CERT_FILE'].present?
-      certificate = File.read(datastore['CERT_FILE'])
+      certificate = File.open(datastore['CERT_FILE'], 'rb', &:read)
```

Comment:
This is ugly, I know, but `rubocop` doesn't let me use a block here:
```
certificate = File.open(datastore['CERT_FILE'], 'rb') { |file| file.read }
```

Reply:
Could the following work?
```suggestion
      certificate = File.binread(datastore['CERT_FILE'])
```